bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -1025,11 +1217,17 @@ func (gcp *Gcp) createGcpServiceAcctSecret(ctx context.Context, client *clientse
 	}
 	saKey, err := iamService.Projects.ServiceAccounts.Keys.Create(name, req).Context(ctx).Do()
 	if err != nil {
-		return fmt.Errorf("Service account key creation error: %v", err)
+		return &kfapis.KfError{
+			Code:    int(kfapis.INTERNAL_ERROR),
```

Comment:
Switch to "INVALID_ARGUMENT"?
We are purely depend on iam API here so might not be responsible for error here.

Reply:
done.